gwi-project/src/api/record.ts

```diff
@@ -0,0 +1,33 @@
+import { BreedT, BreedsT, RandomBreedImageT } from "./types"
+
+export const getBreedImages = async (breed: string) => {
+  const api = await fetch(`https://dog.ceo/api/breed/${breed}/images`)
+  return (await api?.json()) as BreedsT
+}
+
+export const getRandomDogImages = async () => {
+  const api = await fetch("https://dog.ceo/api/breeds/image/random/50")
+  return (await api?.json()) as BreedsT
+}
+
+export const getRandomBreedImage = async (breed: string) => {
+  const api = await fetch(`https://dog.ceo/api/breed/${breed}/images/random`)
+  return (await api?.json()) as RandomBreedImageT
+}
```

Comment:
Why don't you add the return type instead of casting json response?

Reply:
Hello mate,
Thank you so much for the feedback.
I am using TypeScript because I have been programming with this language since the start of my career (both backend and frontend) so I got used to it and I enjoy coding with it. I used emotion because I really like it and we use it at work as well and I believe I am good at it. We also use styled-components but I wanted to easily handle object styles for writing CSS. I also thinking to start learning sass.

Reply:
I did not add the return type because I prefer the structure that way but if you want me to experiment with something else don't hesitate to ask me to do it.